ingress/echoheaders/nginx.conf

```diff
@@ -17,9 +17,6 @@ http {
 
         # Replace '_' with your hostname.
         server_name _;
-        if ($http_x_forwarded_proto = "http") {
```

Comment:
actually can I trouble you to also:
```
mkdir echoheaders-redirect
rsync -a echoheaders/ echoheaders-redirect/
```
and modify that README to say it's forked from echoheaders-1.3?